### PR TITLE
Fixing up #215

### PR DESCRIPTION
**Pull request by philsturgeon**
Just fixing up some conflicts with #215 so we can get this fantastic PR merged in a timely fashion.

### CODE REVIEW

File: example/address.json [hunk 1]
```diff
@@ -1,19 +1 @@
...
-        "postal-code": { "type": "string" },
-        "country-name": { "type": "string"}
-    },
-    "required": ["locality", "region", "country-name"],
-    "dependencies": {
-        "post-office-box": ["street-address"],
-        "extended-address": ["street-address"]
-    }
-}
+{}
```
Comment (handrews) [inline example/address.json]: Why are some of these empty schemas and others deleted entirely?

**Comment by handrews** [comment]
@philsturgeon I'm confused- I had guessed that this would be a PR on top of @retrosight's original PR, but this is a replacement?  What changed?

@Relequestual this is why we really need to be more prompt about merging PRs, it's a huge pain when other work comes in and they get out of sync.

**Comment by philsturgeon** [comment]
I had to resolve some conflicts, which rewrote history and now we've got this PR. Just give it a thumb and we'll be on our way.

**Comment by Anthropic** [comment]
@handrews @philsturgeon was it intentional to break all the examples?
geo, person, card etc... are now all `{}`

**Comment by handrews** [comment]
@Anthropic yes we wanted to stop hosting them as separate files because we are not a schema hosting service.  I checked around and noticed that most of the examples are trivial, and the one tied to an external format (geojson) was actually implementing an outdated draft.  I found I think just one reference to them by googling, and those people had already decided to find more modern/comprehensive schemas elsewhere.

I did not realize that we'd kept the files at all.

**Comment by Anthropic** [comment]
@handrews ok well then maybe we should remove them rather than having them point to empty schema which appears to have caused some confusion for a user that was running test builds off them.

**Pull request merged**